src/UIAutomation/FunctionalTests/UiaOperationAbstractionTests.cpp

```diff
@@ -174,6 +174,37 @@ namespace UiaOperationAbstractionTests
             ElementGetNameTest(true);
         }
 
+        // Asserts that you can get the name of a UiaElement via a property ID.
+        void ElementGetNameViaGetPropertyValueTest(const bool useRemoteOperations)
+        {
+            ModernApp app(L"Microsoft.WindowsCalculator_8wekyb3d8bbwe!App");
+            app.Activate();
+            auto calc = WaitForElementFocus(L"Display is 0");
+
+            auto guard = InitializeUiaOperationAbstraction(useRemoteOperations);
+
+            auto scope = UiaOperationScope::StartNew();
+
+            UiaElement element = calc;
+            auto val = element.GetPropertyValue(UiaPropertyId(UIA_NamePropertyId));
```

Comment:
Add tests verifying `ignoreDefalut` and `useCachedApi` work as expected?

Reply:
I agree there should be tests, but I'm a little bit lost as to how to do this appropriately at the moment.
Testing ignoreDefault=false is easy enough. We just need to fetch a property we know is not implemented, and assert that it is its correct default empty value.
For ignoreDefault=true, we really want to test that fetching a property that is not implemented fails with an exception. Locally, I would assume this test framework has some kind of assertRaisesException, we could wrap around the call to GetPropertyValue, and ensure it does raise an exception. However, remotely, we don't see the exception until scope.resolve is called. 
For useCachedAPI: again locally, we could try fetching a property we know is not already cached, and test for the exception. Remotely, again we won't see this until scope.Resolve.
I'm aware that the lower level remote ops machinary now has support for try blocks. If there are plans to implement try blocks for the higher-level UiaOperationAbstraction, then I think it might be better to wait until that is there, and write the tests using that, which means they can be written the same way for both remote and local.